tests/v2/integration/catalogv2/cluster_repo_test.go

```diff
@@ -935,8 +935,8 @@ func (c *ClusterRepoTestSuite) testClusterRepoRetries(params ClusterRepoParams)
 	cr.Spec.InsecurePlainHTTP = params.InsecurePlainHTTP
 	cr.Spec.GitBranch = "invalid-branch"
 	expoValues := v1.ExponentialBackOffValues{
-		MinWait:    30,
-		MaxWait:    60,
+		MinWait:    60,
+		MaxWait:    120,
```

Comment:
Would bumping these values reduce the chances of test failing? If the goal is to figure out the reasons for test flakiness this may result in lower chance of hitting it (which is both a good (for CI stability) and bad thing (for getting to the bottom of this flake)).

Reply:
Yes you are right. I will remove this commit.